packages/fetchai/agents/confirmation_aea_aw1/aea-config.yaml

```diff
@@ -0,0 +1,56 @@
+agent_name: confirmation_aea_aw1
+author: fetchai
+version: 0.1.0
+description: This agent manages confirmation of registration for Agent World 1
+license: Apache-2.0
+aea_version: '>=0.7.0, <0.8.0'
+fingerprint: {}
+fingerprint_ignore_patterns: []
+connections:
+- fetchai/ledger:0.8.0
+- fetchai/p2p_libp2p:0.12.0
+- fetchai/soef:0.11.0
+contracts: []
+protocols:
+- fetchai/contract_api:0.7.0
+- fetchai/default:0.8.0
+- fetchai/ledger_api:0.6.0
+- fetchai/oef_search:0.9.0
+- fetchai/register:0.1.0
+- fetchai/signing:0.6.0
+- fetchai/state_update:0.6.0
+skills:
+- fetchai/confirmation_aw1:0.1.0
```

Comment:
You mentioned `fetchai/simple_service_registration` in the readme but it does not appear in this.

Reply:
I added it now, thanks